src/main/resources/assets/js/katex-render.js

```diff
@@ -0,0 +1,14 @@
+(function () {
```

Comment:
Could you please add a comment with the source of that script?

Reply:
I amended my commit to include a comment with a link to flexmark wiki where the script come from